### ISSUE TITLE
Shortcut / webapp bug

### ISSUE THREAD

**Issue by tanmaytez** [MM2-0/Kvaesitso#465]
After tapping  "install app to homescreen" option in the browser to place shortcut in homescreen.. it switches to default launcher and when I gesture back or go homescreen it becomes normal( switches to this launcher)

This ain't much annoying, but is this fixable?

**Comment by MM2-0** [comment]
I can't reproduce such behavior, can you check if there are any crash logs (settings > debug > crash reporter).

And is the shortcut still created? And in which launcher?

**Comment by tanmaytez** [comment]
Yes, there are many crash logs and yeah the shortcut is created in favourites.

Default launcher is realme's system launcher.. it just switches or something to default launcher and goes back to kvaesisto launcher when I go home

**Comment by rahaaatul** [comment]
Mine does not even create the shortcut in favorites.


https://github.com/MM2-0/Kvaesitso/assets/86149861/1024830e-3e65-4148-b285-fc264885055f

**Comment by yant0** [comment]
@rahaaatul can i ask what icons youre using?

**Comment by rahaaatul** [comment]
> @rahaaatul can i ask what icons youre using?

Of course! I'm using [Lawnicons](https://github.com/LawnchairLauncher/lawnicons).
